src/js/components/Filter/FilterForm.jsx

```diff
@@ -33,11 +32,10 @@ const FilterForm = ({
       filterElement: true,
     },
   };
-  // Create initialValues from filterFields as empty values
-  const initialValues = Object.keys(filterFields).reduce((acc, key) => {
-    if (!acc[key]) return { ...acc, [key]: '' };
-    return acc;
-  }, { ...defaultValues });
+
+  useEffect(() => {
+    updateFilterParams(defaultValues);
+  }, [defaultValues]);
```

Comment:
First I wrote that:
> I don't like the naming here. You have `defaultValues`, which I think should not change because they are `default` and you expect that it will change.

but then I read the rest of the files. Please add the comment on this `useEffect` that `// Default values can change based on the current location`